enterprise/login-email-password-setup.md

```diff
@@ -0,0 +1,7 @@
+# Login with email and password
```

Comment:
Can we rename this to "Authentication with email and password" and the file to "authentication-email-password-setup.md"?